src/shared/shared.types.ts

```diff
@@ -0,0 +1,10 @@
+// shared types for `Heading` and `Text` component
```

Comment:
nit: same here

Reply:
I am not 100% on that naming ... just kinda slipped through while i added new functions and files .. we can absolutely go with just the file-name without the `shared.` prefix